pkg/kubelet/volumes.go

```diff
@@ -77,12 +77,18 @@ func (kl *Kubelet) mountExternalVolumes(pod *api.BoundPod) (volumeMap, error) {
 	for i := range pod.Spec.Volumes {
 		volSpec := &pod.Spec.Volumes[i]
 
+		podRef, err := api.GetReference(pod)
```

Comment:
+1 to make GetReference not fail on a blank selflink.  @derekwaynecarr any opinion on that?